<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertWidget</class>
 <widget class="QWidget" name="AlertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预警</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AlertWidget {
    background-color: #f5f7fa;
}

QFrame#alertCard {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e4e7ed;
}

QLabel#titleLabel {
    font-size: 18px;
    font-weight: bold;
    color: #303133;
}

QLabel#noAlertLabel {
    font-size: 16px;
    color: #67C23A;
}

QScrollArea {
    border: none;
    background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>⚠️ 天气预警</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="noAlertLabel">
     <property name="text">
      <string>✅ 当前无天气预警</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="alertScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="alertScrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>560</width>
        <height>280</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="alertLayout">
       <property name="spacing">
        <number>12</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
